assets/js/script.js

```diff
@@ -0,0 +1,120 @@
+//******************************DOM OBJECTS******************************
+const newTaskInput = document.getElementById("new-task-input");
+const addTaskBtn = document.getElementById("addBtn");
+const searchBar = document.getElementById("search");
+const table = document.getElementById("table-body");
+const counter = document.getElementById("tasks-counter");
+
+var tasksArray = []; //this array contains all the todo list tasks
```

Comment:
Use ES6 Features:
Take advantage of ES6 features like arrow functions, const, and let for variable declarations to improve code readability and maintainability.